catalog/pipeline/fbc-release/0.3/fbc-release.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: fbc-release
+  labels:
+    app.kubernetes.io/version: "0.3"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton release pipeline to interact with FBC Pipeline
+  params:
+    - name: snapshot
+      type: string
+      description: The Snapshot in JSON format
+    - name: enterpriseContractPolicy
+      type: string
+      description: JSON representation of the EnterpriseContractPolicy
+    - name: fbcFragment
+      type: string
+      description: FBC fragment built by HACBS
+    - name: fromIndex
+      type: string
+      description: Index image (catalog of catalogs) the FBC fragment will be added to
+    - name: overwriteFromIndex
+      type: string
+      description: Boolean indicating if the fromIndex should be overwritten
+    - name: binaryImage
+      type: string
+      description: OCP binary image to be baked into the index image
+    - name: buildTags
+      type: string
+      description: List of additional tags the internal index image copy should be tagged with
+    - name: addArches
+      type: string
+      description: List arches to be added to be built
+    - name: requestUpdateTimeout
+      type: string
+      description: Max seconds to wait until the status is updated
+  workspaces:
+    - name: release-workspace
+  results:
+    - name: requestMessage
+      value: $(tasks.create-internal-request.results.requestMessage)
+    - name: requestReason
+      value: $(tasks.create-internal-request.results.requestReason)
+    - name: requestResults
+      value: $(tasks.create-internal-request.results.requestResults)
+  tasks:
+    - name: verify-enterprise-contract
+      taskRef:
+        name: verify-enterprise-contract
+        bundle: quay.io/hacbs-contract/ec-task-bundle:snapshot
+      params:
+        - name: IMAGES
+          value: $(params.snapshot)
+        - name: SSL_CERT_DIR
+          value: /var/run/secrets/kubernetes.io/serviceaccount
+        - name: POLICY_CONFIGURATION
+          value: $(params.enterpriseContractPolicy)
+        - name: STRICT
+          value: "1"
+    - name: create-internal-request
+      taskRef:
+        name: create-internal-request
+        bundle: quay.io/hacbs-release/task-create-internal-request:0.1
+      params:
+        - name: pipelineRunName
+          value: $(context.pipelineRun.name)
+        - name: fbcFragment
+          value: $(params.fbcFragment)
+        - name: fromIndex
+          value: $(params.fromIndex)
+        - name: overwriteFromIndex
+          value: $(params.overwriteFromIndex)
+        - name: binaryImage
+          value: $(params.binaryImage)
+        - name: buildTags
+          value: $(params.buildTags)
+        - name: addArches
+          value: $(params.addArches)
+        - name: requestUpdateTimeout
+          value: $(params.requestUpdateTimeout)
```

Comment:
I think you could already add line you suggested.
```
runAfter:
  - verify-enterprise-contract
```

Reply:
fixed